packages/react/slot/src/Slot.stories.tsx

```diff
@@ -4,38 +4,184 @@ import { Slot, Slottable } from './Slot';
 export default { title: 'Components/Slot' };
 
 export const WithoutSlottable = () => (
-  <AsCompWithoutSlottable as={Slot}>
```

Comment:
oh wow these hadn't been updated??!

Reply:
Yeah but they didn't matter really, they're just stories showing that `Slot` works so I left them before but changed them now because it definitely made me double take 😅